test/unit/pipeline/number-nonblank.ts

```diff
@@ -0,0 +1,24 @@
+/*
+	File: test/unit/pipeline/number-nonblank.ts
+	cpuabuse.com
+*/
+
+/**
+ * Tests `numberNonBlank`.
+ */
+
+import { strictEqual } from "assert";
+import { processNumberNonBlank } from "../../../src/pipeline/number-nonblank";
+import { animals as input } from "../../input/pipeline";
+import { animals as expected } from "../../expected/pipeline";
+
+// Test for multiple lines
+export function testNumberNonBlank(): void {
+	describe("number-nonblank", function() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['describe' is not defined.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5058624)